explorer/client/src/components/ownedobjects/OwnedObjects.tsx

```diff
@@ -42,24 +47,67 @@ function OwnedObjectStatic({ objects }: { objects: string[] }) {
     return <OwnedObjectView results={results} />;
 }
 
+type IdResultsPair = [string, GetObjectInfoResponse];
+
 function OwnedObjectAPI({ objects }: { objects: string[] }) {
     const [results, setResults] = useState(DATATYPE_DEFAULT);
     const [isLoaded, setIsLoaded] = useState(false);
 
     useEffect(() => {
-        Promise.all(objects.map((objID) => rpc.getObjectInfo(objID))).then(
-            (results) => {
-                setResults(
-                    results.map(({ id, objType, version, data }) => ({
-                        id: id,
-                        Type: objType,
-                        Version: version,
-                        display: processDisplayValue(data.contents?.display),
-                    }))
-                );
-                setIsLoaded(true);
-            }
-        );
+        Promise.all(
+            objects.map(
+                (objID) =>
+                    new Promise<IdResultsPair>((resolve, reject) => {
+                        rpc.getObjectInfo(objID)
+                            .then((res) => {
+                                resolve([objID, res]);
+                            })
+                            .catch((err) => {
+                                reject(err);
+                            });
+                    })
+            )
+        ).then((pairs) => {
+            const results = pairs.map((pair) => {
+                const result = pair[1];
+
+                if (
+                    result.status === 'Exists' &&
+                    isObjectExistsInfo(result.details)
+                ) {
+                    const obj = result.details.object;
+                    return {
+                        id: pair[0],
+                        Type: obj['objType'] as string,
+                        Version: (obj['version'] as number).toString(),
+                        display: processDisplayValue(obj['display'] as string),
```

Comment:
Hmm I don't think these are the fields you will receive from `getObjectInfo` . I got these when calling `GetObjectInfo` for `Coin<Sui>`

```
{
    "jsonrpc": "2.0",
    "result": {
        "status": "Exists",
        "details": {
            "objectRef": {
                "objectId": "8dc6a6f70564e29a01c7293a9c03818fda2d049f",
                "version": 2,
                "digest": "2PfMhH8NlP5eG1rq+NokUASK8HovA1PRlgmksm0QAcQ="
            },
            "object": {
                "contents": {
                    "fields": {
                        "id": {
                            "fields": {
                                "id": {
                                    "fields": {
                                        "id": {
                                            "fields": {
                                                "bytes": "8dc6a6f70564e29a01c7293a9c03818fda2d049f"
                                            },
                                            "type": "0x2::ID::ID"
                                        }
                                    },
                                    "type": "0x2::ID::UniqueID"
                                },
                                "version": 2
                            },
                            "type": "0x2::ID::VersionedID"
                        },
                        "value": 99873
                    },
                    "type": "0x2::Coin::Coin<0x2::SUI::SUI>"
                },
                "owner": {
                    "AddressOwner": "c5206dd02c86a510c4848516229b02addfacbe55"
                },
                "tx_digest": "Zb51CbtIKEbzvwjSRyFoMTVQCo3hG8rjy9vnFEGP4X0="
            }
        }
    },
    "id": 1
}
```